sw/device/silicon_creator/lib/ownership/owner_block.c

```diff
@@ -237,13 +278,31 @@ rom_error_t owner_block_flash_apply(const owner_flash_config_t *flash,
                                              : 0;
   size_t len = (flash->header.length - sizeof(owner_flash_config_t)) /
                sizeof(owner_flash_region_t);
-  if (len > kProtectSlots - kRomExtRegions) {
+  if (len > kProtectSlots) {
+    // We really want `kProtectSlots - kRomExtRegions`, but we have to
+    // accomodate configurations that may have specified ROM_EXT protection
+    // regions.  We deal with that fact in the loop below.
```

Comment:
Similar to the previous comment, this was to accommodate existing chips with an older config.